datasets/loader_clf.py

```diff
@@ -0,0 +1,602 @@
+# ===============================================================================
+# Copyright 2020-2021 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ===============================================================================
+
+import logging
+import os
+from pathlib import Path
+from typing import Any
+
+import numpy as np
+import pandas as pd
+from sklearn.datasets import fetch_openml, load_svmlight_file
+from sklearn.model_selection import train_test_split
```

Comment:
I guess, scikit-learn should be mentioned in readme as requirement for data loading

Reply:
added such requirement to all readme's

Reply:
Add to cuML's readme and check that it's listed in all *_bench/requirements.txt

Reply:
done